frontend/playwright-globalSetup.ts

```diff
@@ -2,23 +2,32 @@ import { chromium } from "@playwright/test";
 import { SKIP_LOGIN, TEST_URL } from "tests/common/constants";
 import { COMMON_PLAYWRIGHT_CONTEXT } from "tests/common/context";
 import featureFlags from "tests/common/featureFlags";
-import { login } from "tests/utils/helpers";
+import { goToPage, login } from "tests/utils/helpers";
 
 module.exports = async () => {
+  const browser = await chromium.launch();
+
+  const browserContext = await browser.newContext({
+    ...COMMON_PLAYWRIGHT_CONTEXT,
+    storageState: featureFlags,
+  });
+
+  const page = await browserContext.newPage();
+
+  await goToPage(undefined, page);
+
+  const apiUrl = await page.evaluate(() => {
```

Comment:
during global setup phase of Playwright, we hit the homepage to detect which API URL the FE app connects to and save that as env variable for subsequent tests to access via helper function `conditionallyRunTests()`